torch/jit/__init__.py

```diff
@@ -1094,6 +1094,12 @@ def _is_new_style_class(cls):
         return ('__dict__' in dir(cls) or hasattr(cls, '__slots__'))
 
 
+def _is_exception(obj):
+    if not inspect.isclass(obj):
```

Comment:
Do we need `if not inspect.isclass(obj):` ?

Reply:
`issubclass` requires a class as the second argument